### ISSUE TITLE
code examples are not working 

### ISSUE THREAD

**Issue by muddlebee** [paritytech/asset-transfer-api#448]
Steps to reproduce : 

1. installed latest depdency from https://www.npmjs.com/package/@substrate/asset-transfer-api
2. ran a few examples and modified the imports from npm packages instead
https://github.com/paritytech/asset-transfer-api/blob/main/examples/WestendToAssetHubXcmV4.ts
https://github.com/paritytech/asset-transfer-api/blob/main/examples/fetchFeeInfo.ts
 
```
import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api';
import { TxResult } from '@substrate/asset-transfer-api/lib/types';
```

getting undefined error below

```
 npx tsx .\fetchFeeInfo.ts
TypeError: Cannot read properties of undefined (reading '1000')
    at parseRegistry (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\node_modules\@substrate\asset-transfer-api\src\registry\parseRegistry.ts:150:17)
    at Registry.set setRegistry (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\node_modules\@substrate\asset-transfer-api\src\registry\Registry.ts:157:32)
    at AssetTransferApi.initializeRegistry (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\node_modules\@substrate\asset-transfer-api\src\AssetTransferApi.ts:426:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async AssetTransferApi.createTransferTransaction (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\node_modules\@substrate\asset-transfer-api\src\AssetTransferApi.ts:209:4)
    at async main (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\fetchFeeInfo.ts:21:14)
Error: TypeError: Cannot read properties of undefined (reading '1000')
    at main (C:\Users\91824\Documents\Codes\swush-me\asset-apis-self\fetchFeeInfo.ts:38:9)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by lrazovic** [comment]
It should be already solved in https://github.com/paritytech/asset-transfer-api/commit/da53859c6ec5213864b409bcbd93943895b13228. We just need to wait for a new release.

**Comment by marshacb** [comment]
Hello @muddlebee, thanks for the issue. I plan to do a release which should fix this today.

**Comment by muddlebee** [comment]
thank you folks :) appreciation the help.

**Comment by marshacb** [comment]
fixed in release 0.4.4
